### PR TITLE
Update staging and add deploy instructions to README

### PR DESCRIPTION
**Pull request by amyrlam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,16 +51,23 @@ View at http://localhost:4200
...
-### Building
+## Staging environment
+
+Latest changes from master can be seen on [ember-api-docs on Heroku](https://dashboard.heroku.com/pipelines/9368de86-caa5-4192-a96d-2ee22dc6ce71). You'll need access to Heroku via the Learning Team. In the "Staging" column, you can check out "Open app" under ember-api-docs-staging.
+
+## Deployment
+
+**Do not use "promote to production" on Heroku.**
+
+After you've checked out the app on Staging above, on [ember-api-docs on Heroku](https://dashboard.heroku.com/pipelines/9368de86-caa5-4192-a96d-2ee22dc6ce71) go to the "Production" column. From the toggle button, select "Deploy a branch".
```
Comment (jenweber) [inline README.md:68]: We have detailed docs that cover this. Please have a read through and see if there's anything that you want to change or add.

```suggestion
See the [DEPLOYING.md](https://github.com/ember-learn/ember-api-docs/blob/master/DEPLOYING.md) guide for instructions.
```

**Review by jenweber — APPROVED** [review]
Other than linking to the deploying guide, I think this is good to go! Thank you!

File: README.md (hunk 1 above)
Comment (amyrlam) [reply README.md:68]: Did not know that was there, thank you! Got it all from chat

**Comment by amyrlam** [comment]
A `yarn run lint` passes...am confused if the CI failure (which doesn't seem to show any details?) is just an off thing, seemed slow today. Will try again later. Or if anyone has insight let me know!

**Comment by amyrlam** [comment]
I think GitHub actions was having issues, done now!

**Pull request merged**